content/200-concepts/100-components/02-prisma-client/260-client-extensions/index.mdx

```diff
@@ -140,3 +140,15 @@ In your code, you can call any of these clients separately, for example `prismaA
 ### Conflicts in combined extensions
 
 When you combine two or more extensions into a single extended client, then the _last_ extension that you declare takes precedence in any conflict. In the example in option 1 above, suppose there is a method called `myExtensionMethod()` defined in `extensionA` and a method called `myExtensionMethod()` in `extensionB`. When you call `prismaAB.myExtensionMethod()`, then Prisma Client uses `myExtensionMethod()` as defined in `extensionB`.
+
+## Type of an extended client
+
+You can infer the type of an extended Prisma Client instance using the [`ReturnType`](https://www.typescriptlang.org/docs/handbook/utility-types.html#returntypetype) as follows:
```

Comment:
Return type is still mentioned, do you think it would be work adding an example for the two cases? With function and without.